src/bind.ts

```diff
@@ -1,5 +1,31 @@
 import { UnbindFn, Binding } from './types';
 
+// ========================
```

Comment:
I like this solution, but unfortunately not all events on an event target have an object property event handler (eg `DOMContentLoaded`)

Reply:
Yes, not all events have handlers. However, if the handler isn't present on the target, we fall back to the `Event` type, so there shouldn't be any problems.